enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -27,32 +29,34 @@ def now():
     return UTC.localize(datetime.utcnow())
 
 
-class TimeStampedModelWithUuid(TimeStampedModel):
```

Comment:
why did you remove this? i presume we're going to want uuids and timestamps on most stuff? this makes it easier/consistent?

Reply:
It might be worth adding a `TimeStampedModelWithUuid` model to https://github.com/openedx/edx-django-utils